libnetwork/endpoint.go

```diff
@@ -891,6 +891,7 @@ func (ep *Endpoint) sbLeave(ctx context.Context, sb *Sandbox, force bool) error
 		log.G(ctx).WithError(err).Warn("Failed to clean up resolver info on container disconnect")
 	}
 
+	etcHostsAddrs := ep.getEtcHostsAddrs()
 	if err := sb.clearNetworkResources(ep); err != nil {
```

Comment:
This `ep.getEtcHostsAddrs()` call is put deliberately before the `sb.clearNetworkResources(ep)` here?

If that's the case, it could warrant a comment to prevent someone (me!? 😂) from moving it closer to the `sb.deleteHostsEntries(etcHostsAddrs)` call at line 912.

Reply:
Done.